scripts/NewTeamsAlert/src/helper/sendAlert.js

```diff
@@ -0,0 +1,63 @@
+// Function to send adaptive cards
+import dotenv from "dotenv";
+dotenv.config(); // Load environment variables from .env file
+
+import template from "../adaptiveCards/adaptive-card-template.json" assert { type: "json" };
```

Comment:
what is the need of assert here

Reply:
Used Assert to avoid this TypeError [ERR_IMPORT_ASSERTION_TYPE_MISSING], since I am using type : module in package.json and template.json, should I implement this in a different way?